src/http-auth-interceptor.js

```diff
@@ -5,6 +5,12 @@
  * (c) 2012 Witold Szczerba
  * License: MIT
  */
+
+/* commonjs package manager support (eg componentjs) */
+if (typeof module !== "undefined" && typeof exports !== "undefined" && module.exports === exports){
```

Comment:
wouldn't it be simpler to write without those `typeof`s?
`if (module !== undefined && exports !== undefined && module.exports === exports) {...}`